packages/build/src/steps/plugin.js

```diff
@@ -108,7 +108,9 @@ export const firePluginStep = async function ({
     })
     return { newError }
   } finally {
-    await unpipePluginOutput(childProcess, logs, listeners, standardStreams)
+    if (!isTrustedPlugin(pluginPackageJson?.name) || listeners) {
```

Comment:
I don't this code well enough to be sure why presence of `listeners` means we should call the unpipe method, but I trust you 😄